modules/mgmt-power-remediation-baremetal-about.adoc

```diff
@@ -47,6 +50,23 @@ The remediation process operates as follows:
 If the power operations did not complete, the bare metal machine controller triggers the reprovisioning of the unhealthy node unless this is a control plane node or a node that was provisioned externally.
 ====
 
+[id="mgmt-understanding-metal3-remediation-process_{context}"]
+== Understanding the metal3-based remediation process
+
+The remediation process operates as follows:
+
+. The MachineHealthCheck (MHC) controller detects that a node is unhealthy.
+. The MHC creates a metal3 remediation custom resource for the metal3 remediation controller which requests to power-off the unhealthy node.
```

Comment:
```suggestion
. The MHC creates a metal3 remediation custom resource for the metal3 remediation controller, which requests to power-off the unhealthy node.
```

Reply:
Thanks for your feedback @adellape . I have applied the fix.